src/App.tsx

```diff
@@ -8,17 +8,28 @@ import PersonaliaBoks from './components/personalia-boks';
 import { Registrering } from './components/registreringsInfo';
 import { Ytelser } from './components/ytelserinfo';
 import { Chips } from '@navikt/ds-react';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import { useChips } from './data/api/fetch';
+import { LagreChips, NullstillChips } from './components/felles/chipsknapp';
 
 export interface AppProps {
     fnr: string;
     enhet?: string;
 }
 
 const App = (props: AppProps) => {
-    const informasjonsboksAlternativer = ['CV', 'Jobbønsker', 'Oppfølging', 'Personalia', 'Registrering', 'Ytelser'];
+    const chipsData = useChips();
```

Comment:
Flisespikking: "chips" er et UI-konsept og sier ikke noe interessant om hva som ligger bak/hva det brukes til. Et bedre navn er kanskje:

```suggestion
    const overblikkFilter = useOverblikkFilter();
```

😄